File: output/ChatGPT/codenet/Java/C++/s066389476.cpp
#include <iostream>
#include <string>

void solve() {
    std::string a;
    std::cin >> a;
    
    if (a[0] == a[1] || a[1] == a[2] || a[2] == a[0]) {
        std::cout << "No" << std::endl;
    } else {
        std::cout << "Yes" << std::endl;
    }
}

int main() {
    solve();
    return 0;
}

File: output/CodeLlama/codenet/Java/Python/s066389476.py
import sys

def solve(sc):
    a = sc.readline().strip()
    if a[0] == a[1] or a[1] == a[2] or a[2] == a[0]:
        print('No')
    else:
        print('Yes')

if __name__ == '__main__':
    solve(sys.stdin)